pkg/webui/account/components/oauth-client-form/index.js

```diff
@@ -156,6 +184,40 @@ const OAuthClientForm = props => {
           />
         </>
       )}
+      {update && mayEditBasicInformation && (
+        <>
+          <Form.SubTitle title={sharedMessages.contactInformation} className="mb-cs-s" />
+          <Notification small warning content={m.contactWarning} />
+          <CollaboratorSelect
+            name="administrative_contact"
+            title={sharedMessages.adminContact}
+            placeholder={sharedMessages.contactFieldPlaceholder}
+            entity={'client'}
+            entityId={clientId}
+            encode={encodeContact}
+            decode={decodeContact}
+          />
+          <Message
+            content={m.adminContactDescription}
+            component="p"
+            className="mt-cs-xs tc-subtle-gray"
+          />
+          <CollaboratorSelect
+            name="technical_contact"
+            title={sharedMessages.technicalContact}
+            placeholder={sharedMessages.contactFieldPlaceholder}
+            entity={'client'}
+            entityId={clientId}
+            encode={encodeContact}
+            decode={decodeContact}
```

Comment:
```suggestion
            decode={decodeContact}
            required
```
At least it's currently not possible to submit these forms the selector empty.